pss/keystore.go

```diff
@@ -180,26 +179,26 @@ func (ks *KeyStore) processSym(envelope *whisper.Envelope) (*whisper.ReceivedMes
 }
 
 // Attempt to decrypt, validate and unpack an asymmetrically encrypted message.
-// If successful, returns the unpacked whisper ReceivedMessage struct
+// If successful, returns the unpacked receivedMessage struct
 // encapsulating the decrypted message, and the byte representation of
 // the public key used to decrypt the message.
 // It fails if decryption of message fails, or if the message is corrupted.
-func (ks *Pss) processAsym(envelope *whisper.Envelope) (*whisper.ReceivedMessage, string, PssAddress, error) {
+func (ks *Pss) processAsym(envelope *envelope) (*receivedMessage, string, PssAddress, error) {
```

Comment:
once we are at that, can we change the receiver name to p

Reply:
Changed in commit 49a3b06